src/api/controllers/popularArticleController.js

```diff
@@ -0,0 +1,36 @@
+
+import { sequelize } from '../models';
+import status from '../../helpers/constants/status.codes';
+
+/**
+ * containing all user's
+ * profile model controller(update, getUserProfile, read)
+ * @export
+ * @class ProfileController
+ */
+export default class PopularController {
+  /**
+         *
+         * get 5 mostly read article in the database
+         * @static
+         * @param {*} req The request Object
+         * @param {*} res The response Object
+         * @memberof PopularController
+         * @return {Object} The response Object
+         */
+  static async getAll(req, res) {
+    const resp = await sequelize.query(`SELECT a.title, a.slug, a.description, a."createdAt", a."coverImage", a."readTime", COUNT(r."articleSlug") slug_count, c.name category, u."firstName", u."lastName", u.username, u."profileImage"
+    FROM "Articles" a
+    INNER JOIN "Reads" r ON r."articleSlug" = a.slug
+    LEFT JOIN "Categories" c ON c.id = a.category
+    INNER JOIN "Users" u ON u.id = a.author
+    GROUP BY a.title, a.slug,  a.description,  a."coverImage", c.name, a."createdAt", u."firstName", u."lastName", u.username, a."readTime", u."profileImage"
```

Comment:
@MCFrank16 why are you doing a `GROUP BY` here?

Reply:
here I was just Grouping everything up so that I can be able to count and order to get the most popular article

Reply:
I'm curious to know if the query works as expected. Do you have time to join a quick call?